lib/game.rb

```diff
@@ -2,8 +2,6 @@ class Game
 
   # take_turns method
```

Comment:
yay this is done!

Reply:
🎉 Snaps to your big brain moments on this!! 🎉